src/tuvx/tuvx.cpp

```diff
@@ -152,4 +159,17 @@ namespace musica
     return radiator_map;
   }
 
+  void TUVX::Run(const double solar_zenith_angle, const double earth_sun_distance,
+                 double * const photolysis_rate_constants, double * const heating_rates, Error * const error)
+  {
+    *error = NoError();
+    int error_code = 0;
+    double sza_degrees = solar_zenith_angle * 180.0 / M_PI;
```

Comment:
Can't find the definition of `M_PI` within the repository. Where is it defined?

Reply:
Well, I thought it was part of the C standard but apparently it's not, it's just commonly included in `math.h`. I switched this to use `std::numbers::pi` instead